x-pack/plugins/cloud_security_posture/server/routes/status/status.ts

```diff
@@ -51,12 +57,22 @@ const getHealthyAgents = async (
 };
 
 const calculateCspStatusCode = (
-  hasFindings: boolean,
+  indicesStatus: {
+    findingsLatestIndexStatus: IndexStatus;
+    findingsIndexStatus: IndexStatus;
+    scoreIndexStatus: IndexStatus;
+  },
```

Comment:
_nit_
```suggestion
  indicesStatus: {
    findingsLatest: IndexStatus;
    findings: IndexStatus;
    score: IndexStatus;
  },
```
to me `indicesStatus.score` is clear the `score` is an index status (plus we assign it that type)